.prompt_templates/ui/generate_ui_from_figma.md

```
@@ -0,0 +1,48 @@
+
+2. Search all strings in the [FIGMA_LINK] and add the appropriate strings to lib/resource/l10n/intl_ja.arb (no need to translate to other languages and do not add strings that can be dynamic data), later when coding UI use l10n.<key>
+
+3. Search all images in the Figma design and check assets/images folder:
+
+   - If matching file exists: Use it with CommonImage(lib/ui/component/ui_kit/common_image.dart) for next step
+     - For .svg files: Use `CommonImage.svg()` constructor
+     - For .png files: Use `CommonImage.asset()` constructor
+   - If no matching file exists: Create empty image file with corresponding name in assets/images folder (do not create images in the status bar)
+
+4. Based on the data from the get_code command above, complete the {PascalCaseScreenName}Page, {PascalCaseScreenName}ViewModel, {PascalCaseScreenName}State classes
+  - For {PascalCaseScreenName}Page, follow the instructions:
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)